src/game.js

```diff
@@ -27,7 +27,7 @@ export class Game {
   getNewQuiz() {
     if (this.#userAnswers.length === this.#numberOfQuestions) {
       this.#publishGameEndEvent(this.#score);
-      return;
+      this.#saveGameResultsInLocalStorage();
```

Comment:
We still need the `return` here, otherwise the rest of `getNewQuiz()` keeps on executing and we generate another quiz after the game has ended 😅

Reply:
`return `after `this.#saveGameResultsInLocalStorage();` right?